tests/functional/test_patient_access.py

```diff
@@ -115,15 +158,15 @@ async def test_patient_access_nhsd_patient_header_sent_downstream(
                         "path": "/telecom/0",
                         "value": {
                             "id": telecom_id,
-                            "system": "email",
-                            "use": "work",
-                            "value": generate_random_email_address(),
+                            "system": "phone",
+                            "use": "mobile",
+                            "value": generate_random_phone_number(),
```

Comment:
What was the reasoning behind this change?

Reply:
It was test data related, the patch is a replace, the record was in an incorrect state for this to work. We can't use a different record, as it needs to match what's set up for the authentication, but perhaps a different kind of patch would be more flexible.